devel/abseil/Portfile

```diff
@@ -1,16 +1,16 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.1
-PortGroup           github 1.0
-PortGroup           legacysupport 1.1
+PortGroup           cmake                       1.1
+PortGroup           github                      1.0
+PortGroup           legacysupport               1.1
```

Comment:
Do we really need to mess with this?

Some maintainers would strongly prefer that we leave their spacing alone. Please and Thanks.  😉

Reply:
Acknowledged and reverted.